tests/conftest.py

```diff
@@ -35,9 +35,12 @@ def payload(func, *args, **kwargs):
 
 @pytest.fixture
 def amqp_direct_url(request) -> URL:
-    return URL(
+    amqp_url = URL(
         os.getenv("AMQP_URL", "amqp://guest:guest@localhost"),
     ).update_query(name=request.node.nodeid)
+    if amqp_url.port is None:
+        amqp_url = amqp_url.with_port(5672)
```

Comment:
It's not needed cause https://github.com/mosquito/aiormq/blob/master/aiormq/connection.py#L84-L85

Reply:
WAT? That's resolve stuck problem in CI. https://github.com/mosquito/aio-pika/pull/340#issuecomment-673422926

Reply:
@mosquito merge this PR?